File: sem3/map/Lab/3/toylanguage/src/main/java/com/company/Views/Commands/RunProgramCommand.java
package com.company.Views.Commands;

import com.company.Controller.Controller;
import com.company.Views.Command;

public class RunProgramCommand extends Command {
    private Controller controller;

    public RunProgramCommand(String name, String description, Controller controller) {
        super(name, description);
        this.controller = controller;
    }

    @Override
    public void execute() {
        this.controller.executeAllProgram();
    }
}

File: sem3/map/Lab/toylanguage_C#/ToyLanguage/ToyLanguage/Views/Commands/RunProgramCommand.cs
using ToyLanguage.Controller;

namespace ToyLanguage.Views
{
    public class RunProgramCommand : CommandBase
    {
        public RunProgramCommand(ToyLanguageController controller, string name = "run", string description = "Run program.") : base(controller, name, description)
        {
        }

        public override void Execute()
        {
            _controller.ExecuteProgram();
        }
    }
}